hasktorch/src/Torch/Functional.hs

```diff
@@ -2,10 +2,35 @@
 {-# LANGUAGE MultiParamTypeClasses #-}
 
 module Torch.Functional (
-    module Torch.Functional
-  , module Torch.Functional.Internal
```

Comment:
Internal is not exported.
When we need some functions like fft, do we add the test and export it at the time?

Reply:
@junjihashimoto what I've done is add the re-export for the function explicitly in Torch.Functional. The re-exports serve as a whitelist -

https://github.com/hasktorch/hasktorch/pull/277/files#diff-6f6f611002067ebc1c3697bd36244d43